code/modules/client/preference/preferences.dm

```diff
@@ -1797,6 +1803,32 @@ GLOBAL_LIST_INIT(special_role_times, list( //minimum age (in days) for accounts
 
 						flavor_text = msg
 
+				if("ipcloadouts")
+					var/choice
+					var/datum/robolimb/R = new()
+					var/robolimb_companies = list()
+					var/rparts = list("chest", "groin", "head", "r_arm", "r_hand", "r_leg", "r_foot", "l_leg", "l_foot", "l_arm", "l_hand")
+					for(var/comp in typesof(/datum/robolimb))	//This loop populates a list of companies that shells
+						R = new comp()
+						if(R.has_subtypes != 0)					//No Monitor Models
```

Comment:
This is where !=0 means something other than >0. 0 means 'model' in the robolimbs code (previously null). 

We don't want the monitor companies populating our robolimb_companies list. If they show up and we select one, the dmi file associated with the company for the robolimb (a monitor) does not have sprites for all of the limbs. We end up with invisible sprites. 

![image](https://user-images.githubusercontent.com/69871346/104680534-14f04e00-56be-11eb-8c72-d78aa0a8438c.png)